includes/namespace.php

```diff
@@ -190,7 +237,9 @@ function remove_site( $error, $site ) {
  * @return void
  */
 function deactivate( $network_wide ) {
-	if ( version_compare( PHP_VERSION, WEBSTORIES_MINIMUM_PHP_VERSION, '<' ) ) {
+	global $web_stories_compatibility;
+
+	if ( ! $web_stories_compatibility->check_php_version() ) {
```

Comment:
We can also do more checks here.

Reply:
I actually think we don't need a PHP version check here at all.

Plugin activation fails if the PHP version is too low.

So when you are able to deactivate the plugin it actually means you already run it successfully on a correct PHP version.

In other words, there's no scenario where this happens.

Reply:
Agreed. I couldn't workout why this was there in the first place. Should I remove it?

Reply:
Should i remove this check?

Reply:
Let's remove it

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/4930/commits/624efa9570bad1d24353f845246306e1df662e00